src/aatoolbox/datasources/usgs/usgs_ndvi.py

```diff
@@ -0,0 +1,851 @@
+"""Class to download and process USGS FEWS NET NDVI data.
+
+Data is downloaded from the `USGS FEWS NET data portal
```

Comment:
This is all duplicated in `docs/` correct? I would remove it from here then, otherwise it will be impossible to maintain them both. Or just keep the bare minimum here.

Reply:
Is there a way to reference the additional documentation that is elsewhere?

Reply:
You could just link directly to the read the docs page. Maybe we should start doing that for all the data sources.